src/main/java/practice/CandidateValidator.java

```diff
@@ -1,5 +1,24 @@
 package practice;
 
-public class CandidateValidator {
-    //write your code here
+import java.util.function.Predicate;
+import model.Candidate;
+
+public class CandidateValidator implements Predicate<Candidate> {
+    private final int minYearIn = 10;
```

Comment:
same for other constant
read about constant rules
```suggestion
    private static final int MIN_YEARS_IN = 10;
```

Reply:
If I make such a name then it does not pass in tests

Member name 'MIN_YEARS' must match pattern '^[a-z][a-z0-9][a-zA-Z0-9]*$'. [MemberName]

Reply:
I know about the rule for constant names. But tests are not executed if I write according to the rules. Can you explain in more detail, maybe I'm doing something wrong?

Reply:
![изображение_2025-06-27_011756854](https://github.com/user-attachments/assets/d01717aa-fcfa-404b-b2d4-248842d9ecf3)